Models/BusinessLogic.cs

```diff
@@ -80,29 +81,63 @@ public async Task SkipLogin() {
 
 
     #region Quizzes
+    public QuizManager? EditQuizManager { get; set; }
+
+    public QuizManager? getEditQuizManager() {
+        return (EditQuizManager != null) ? EditQuizManager : null;
+    }
+    public void SetEditQuestionQuizManager(QuizManager quizManager) {
+        EditQuizManager = quizManager;
+    }
```

Comment:
Could getEditQuizManager() and SetEditQuizManager be done with just the property